### PR TITLE
Adding advanced installation

### PR DESCRIPTION
**Pull request by axsaucedo**
Adding advanced installation pagess, which provides detailed steps on all of the ecosystem extensions for Seldon Deploy, and extended the bitbucket instructions to also cover github.

### CODE REVIEW

File: content/en/docs/Getting Started/Installation/Advanced Installation/Advanced GitOps/_index.md [hunk 1]
```diff
@@ -173,7 +205,31 @@ echo "Argocd app created for $REPONAME-$namespace"
...
+
+It may be necessary to also select `Skip certificate verification` option.
+
+##### Github
+
+First we need to configure the webhook that will be sending updates to the Seldon Deploy endpoint:
+
+```sh
+whIp=$(kubectl get service seldon-deploy-webhook -o jsonpath='{.status.loadBalancer.ingress[*].ip}')
+curl -u ${GIT_USER}:${GIT_TOKEN} -v -H "Content-Type: application/json" -X POST -d "{\"name\": \"web\", \"active\": true, \"events\": [\"*\"], \"config\": {\"url\": \"http://${whIp}/api/git-webhook\", \"content_type\": \"json\" }}" https://${GIT_API}/repos/${GIT_USER}/${REPONAME}/hooks
```
Comment (RafalSkolasinski) [inline content/en/docs/Getting Started/Installation/Advanced Installation/Advanced GitOps/_index.md]: Could be good to reformat this such that it spreads over multiple lines for better readability.

File: content/en/docs/Getting Started/Installation/Advanced Installation/Advanced GitOps/_index.md [hunk 2]
```diff
@@ -173,7 +205,31 @@ echo "Argocd app created for $REPONAME-$namespace"
...
+```
+
+Then we set up the webhook that will be sending the updates to ArgoCD.
+
+```sh
+curl -u ${GIT_USER}:${GIT_TOKEN} -v -H "Content-Type: application/json" -X POST -d "{\"name\": \"web\", \"active\": true, \"events\": [\"*\"], \"config\": {\"url\": \"https://${ARGOCDURL}/api/webhook\", \"content_type\": \"json\", \"secret\": \"${ARGOCDINITIALPASS}\", \"insecure_ssl\": \"1\" }}" https://${GIT_API}/repos/${GIT_USER}/${REPONAME}/hooks
```
Comment (RafalSkolasinski) [inline content/en/docs/Getting Started/Installation/Advanced Installation/Advanced GitOps/_index.md]: Same.

**Comment by RafalSkolasinski** [comment]
Suggest to merge https://github.com/SeldonIO/seldon-deploy-docs/pull/106 as well as it contains relevant gitops fixes.

**Comment by axsaucedo** [comment]
thanks for the heads up @RafalSkolasinski added the changes suggested on #106 as well

**Pull request merged**